cdc/processor/sinkmanager/manager.go

```diff
@@ -119,6 +122,11 @@ type SinkManager struct {
 
 	// Metric for table sink.
 	metricsTableSinkTotalRows prometheus.Counter
+
+	// To control the error retry.
+	lastInternalError  error
```

Comment:
It's better to put them into a new struct, and put the new struct here.
The field name should be `sinkRetry`, because we will add retry for redo later.